python/taichi/lang/ops.py

```diff
@@ -399,108 +627,218 @@ def random(dtype=float):
 
 
 @binary
-def add(a, b):
-    """The add function.
+def add(x1, x2):
+    """Add arguments element-wise.
 
     Args:
-        a (Union[:class:`~taichi.lang.expr.Expr`, :class:`~taichi.lang.matrix.Matrix`]): A number or a matrix.
-        b (Union[:class:`~taichi.lang.expr.Expr`, :class:`~taichi.lang.matrix.Matrix`]): A number or a matrix.
+        x1, x2 (Union[:class:`~taichi.lang.expr.Expr`, :class:`~taichi.lang.matrix.Matrix`]): \
+            The arguments to be added. If both x1, x2 are matrices they must have the same shape. \
+                For a scalar and a matrix the scalar will be added element-wise with the matrix.
 
     Returns:
-        sum of `a` and `b`.
+        The sum of `x1` and `x2`, element-wise. This is a scalar if both `x1` and `x2` are scalars.
+
+    Example::
+
+        >>> @ti.kernel
+        >>> def main():
```

Comment:
nit: better not to name them as `main` :P

Reply:
renamed `main` to `test`